custom_components/hwam_stove/translations/de.json

```diff
@@ -0,0 +1,202 @@
+{
+  "config": {
+    "step": {
+      "init": {
+        "data": {
+          "name": "Name",
+          "host": "Host",
+          "id": "ID"
+        }
+      }
+    },
+    "error": {
+      "already_configured": "Host bereits konfiguriet",
+      "id_exists": "ID existiert bereits",
+      "cannot_connect": "Kann nicht mit Host verbinden"
+    }
+  },
+  "device": {
+    "hwam_remote_device": {
+      "name": "HWAM Raum Temperatur Sensor"
+    },
+    "hwam_stove_device": {
+      "name": "HWAM Smart Ofen"
+    }
+  },
+  "entity": {
+    "binary_sensor": {
+      "refill_alarm": {
+        "name": "Nachlegen nötig",
+        "state": {
+          "off": "Nein",
+          "on": "Ja"
+        }
+      },
+      "maintenance_alarms": {
+        "name": "Wartung"
+      },
+      "maintenance_alarms_backup_battery_low": {
+        "name": "Backup Batterie"
+      },
+      "maintenance_alarms_o2_sensor_fault": {
+        "name": "O2 sensor"

```

Comment:
`O2` was recently changed to `Oxygen` in the English (and Dutch) translation(s). Can we do something similar here?

Reply:
`Sauerstoff` has my preference over `O2` for these sensors.
I got the names for these sensors from an older version of the IHS Android app. If I remember correctly these sensors were named in a "hidden" debug menu, so maybe the names are not 100% accurate. Is the IHS software open source and/or reverse engineerable?